controllers/projects/delete.js

```diff
@@ -1,8 +1,38 @@
 'use strict'
 
+var db = require('../../database.js');
+
 module.exports = function(app) {
 
-	app.get('/projects/delete/:id', function(req, res) {
-		res.send('Not done yet.');
+	app.post('/projects/delete/:id', function(req, res) {
```

Comment:
`app.get` was used because usually we're going to use an HTML anchor tag (`<a href="URL">delete</a>`) as the UI element for deleting. `app.post` means we'd have to use something like `<form method="post" action="URL"><input type="submit" value="delete"></form>`